### ISSUE TITLE
Fulcrum Jekyll Footer Copyright Text Has Insufficient Color Contrast

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#609]
FOOTER COPYRIGHT: Element has insufficient color contrast of 2.00 (foreground color: #565b63, background color: #042d5f, font size: 15.0pt, font weight: normal)

`body > .page-footer > .footer-copyright.right-align > .container`
`<div class="container"> © 2016 Regents of the University of Michigan</div>`

- [ ] 1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)
Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.
